bin/maya.bat

```diff
@@ -23,9 +23,11 @@ if "%MINDBENDER_ASSET%"=="" goto :missing_set_asset
 :: If user forgets to include task with "maya"..
 if "%1"=="" goto :missing_task
 
-set MINDBENDER_WORKDIR=%CD%\work\%1\%USERNAME%\maya
+if not exist "%1" goto :missing_taskfolder
+
+set MINDBENDER_WORKDIR=%CD%\%1\%USERNAME%\maya
 if Not exist %MINDBENDER_WORKDIR% (
-	echo Creating new task "%1"..
+	echo Creating folder for task "%1"..
```

Comment:
it now says:
> Creating new task 
But for comedy sake, I will just leave this here:

I TOLD YOU SO ❗️